app/jenkins-for-jira-ui/src/api/fetchGlobalPageUrl.ts

```diff
@@ -1,5 +1,20 @@
 import { invoke } from '@forge/bridge';
 
+type FetchSiteNameProps = {
+	withProtocol: boolean
+};
+
```

Comment:
/nit - we refer to is as siteName and siteUrl, can we rename the function to fetchSiteUrl

Reply:
I'll create a ticket to standardise this. It started out with Dom calling it a site url then later switching to siteName, hence the naming here.